tla/governance-ledger/Governance.tla

```diff
@@ -95,12 +95,29 @@ Apalache caveats to keep in mind:
 Optimizations
 *************
 
+\* TODO: this documentation should be updated to not refer to Merge_Maturity_Of_Neuron
 The model deviates from the Rust implementation by (among other things) employing some optimizations.
 In particular, validity checks (e.g., in Merge_Maturity_Of_Neuron) over method parameters are often 
 moved to the start of the action, such that the choice of the parameters is restricted immediately, 
 rather adding an "await" at the same point as where the implementation performs its checks.
 This shortens the time needed by TLC to figure out that an action is disabled in a particular state.
 
+
+***********
+Deviations
```

Comment:
Could you add the merge_neuron differences here?

Reply:
done